bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -24,8 +24,16 @@
   {{ _('Did you know? Mozilla — the maker of Firefox — fights to keep the Internet a global public resource, open and accessible to all.') }}
 {% endblock %}
 
+{% set cop = request.GET.get('v', None) %}
+
 {% block body_id %}home{% endblock %}
-{% block body_class %}lang-{{ LANG }}{% endblock %}
+{% block body_class %}lang-{{ LANG }} var-{{cop}}{% endblock %}
+
+{% block experiments %}
+  {% if switch('statement-04-2018', ['en-US']) %}
```

Comment:
I'd recommend a separate switch to toggle the traffic cop experiment without toggling the whole takeover. So we could turn off the A/B test and keep the default variant active.

Reply:
Oh I also get an error on the last `)` here. Should just be `{% if switch('statement-04-2018', ['en-US'] %}`